js/src/admin/utils/getCategorizedExtensions.js

```diff
@@ -5,23 +5,17 @@ export default function getCategorizedExtensions() {
     const extension = app.data.extensions[id];
     let category = extension.extra['flarum-extension'].category;
 
-    if (!extension.extra['flarum-extension'].category) {
-      category = 'other';
-    }
-
     // Wrap languages packs into new system
-    if (extension.extra['flarum-locale']) {
+    if (extension.extra['flarum-locale'] && !category) {
```

Comment:
Maybe this should be applied to ALL language packs regardless of what they indicate as their category? Not 100% sure though

Reply:
That would make sense to me, not sure why they would be anything else.